site/content/contribute/mobile/build-your-own/android.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Build the Android App"
+date: 2018-05-20T11:35:32-04:00
+weight: 4
+subsection: Build Your Own App
+---
+
+Android requires that all apps be digitally signed with a certificate before they can be installed, so to distribute your Android application via the Google Play Store, you'll need to generate a signed release APK.
```

Comment:
Reword to:

To distribute your Android application via the Google Play Store, you'll need to generate a signed release APK as Android requires all apps to be digitally signed with a certificate before they can be installed.